src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -55,12 +55,12 @@ export default class Project extends React.Component {
             <Button onClick={runQuery}>{t('Update')}</Button>
           </Box>
         </DropdownLink>
-      </StyledProject>
+      </Flex>
     );
   }
 }
 
-const StyledProject = styled(Flex)`
+export default styled(MultipleProjectSelector)`
```

Comment:
Similar to work done in https://github.com/getsentry/sentry/pull/8911 - we should make sure default export is a styled component so that we can import default module and expect it to be a SC (i.e. if we wanted to use component as a selector).